src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -366,14 +422,63 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
                     scene.pointerY = this._screenCoordinatesRef.y;
                 }
             }
-            controllerData.pick = this._scene.pickWithRay(controllerData.tmpRay, this._scene.pointerMovePredicate || this.raySelectionPredicate);
 
-            const pick = controllerData.pick;
+            let pick = null;
+            let hoverAtOrigin = false;
+            let hoverRange = false; 
+            if (controllerData.hoverIndexMeshTip) {
+                let hoverPickInfo = this._pickWithMesh(controllerData.hoverIndexMeshTip, false, this.nearPickPredicate);

```

Comment:
These could probably be combined, to reduce the number of pickWithMesh calls.
(mesh)=>{return this.nearPickPredicate(mesh) || this.nearGrabPredicate(mesh);}